kyo-prelude/shared/src/main/scala/kyo2/kernel/Effect.scala

```diff
@@ -165,6 +162,51 @@ object Effect:
             handle3Loop(v, Context.empty)
         end apply
 
+        private[kyo2] inline def partial[I1[_], O1[_], E1 <: Effect[I1, O1], I2[_], O2[_], E2 <: Effect[I2, O2], I3[_], O3[_], E3 <: Effect[
```

Comment:
This is designed specifically for the needs of `IOTask`. You can see its use here: https://github.com/getkyo/kyo/blob/82890d80159f366e80fe4a3f81b8c551129c9b61/kyo-core2/shared/src/main/scala/kyo2/scheduler/IOTask.scala#L39